eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -1073,4 +1073,22 @@ function Update-ReleaseStatusInReleasePlan($releasePlanWorkItemId, $status, $ver
     Write-Host "Updating Release Plan [$releasePlanWorkItemId] with status [$status] for language [$LanguageShort]."
     $workItem = UpdateWorkItem -id $releasePlanWorkItemId -fields $fields
     Write-Host "Updated release status for [$LanguageShort] in Release Plan [$releasePlanWorkItemId]"
+}
+
+function Update-PullRequestInReleasePlan($releasePlanWorkItemId, $pullRequestUrl, $status, $languageName)
+{
+    $devopsFieldLanguage = Get-LanguageDevOpsName -LanguageShort $languageName
+    if (!$devopsFieldLanguage)
+    {
+        Write-Host "Unsupported language to update release plan, language [$languageName]"
+        return $null
+    }
+
+    $fields = @()
+    $fields += "`"SDKPullRequestFor$($devopsFieldLanguage)=$pullRequestUrl`""
+    $fields += "`"SDKPullRequestStatusFor$($devopsFieldLanguage)=$status`""
```

Comment:
is `SDK PR status for ...` is a text field? or it has specific values

Reply:
It's a text field so it can say "Failed to generate"